frontend/src/Music.jsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import {
+  Dimensions, Switch, View,
+} from 'react-native';
+import { Picker } from '@react-native-picker/picker';
+import { LinearGradient } from 'expo-linear-gradient';
+import {
+  AmbianceTitle,
+} from './Themes';
+
+function Music() {
+  const [musicOn, setMusicOn] = React.useState(true);
+  const [selectedSong, setSelectedSong] = React.useState();
+
+  return (
+    <LinearGradient
+      colors={['#F9F6FF', '#CFDFF7']}
+      style={{
+        flex: 1,
+        padding: 15,
+        alignItems: 'center',
+      }}
+    >
+      <View style={{
+        flexDirection: 'row',
+        marginBottom: 30,
+        marginTop: 10,
+        justifyContent: 'center',
+      }}
+      >
+        <AmbianceTitle>Music</AmbianceTitle>
+        <Switch
+          onValueChange={(value) => setMusicOn(value)}
+          value={musicOn}
+          style={{ transform: [{ scaleX: 1.4 }, { scaleY: 1.4 }] }}
+        />
+      </View>
+      <Picker
+        selectedValue={selectedSong}
+        onValueChange={(itemValue) => setSelectedSong(itemValue)}
+        style={{ width: Dimensions.get('window').width - 60 }}
+      >
```

Comment:
I think it can be jarring for the user in the case when the switch is off but the user can still scroll through the different selections. I'd suggest using the `enabled` prop to prevent this case.
```suggestion
      <Picker
        selectedValue={selectedSong}
        onValueChange={(itemValue) => setSelectedSong(itemValue)}
        style={{ width: Dimensions.get('window').width - 60 }}
        enabled={musicOn}
      >
```